crates/iota-genesis-builder/src/lib.rs

```diff
@@ -118,6 +114,8 @@ pub struct Builder {
     genesis_stake: GenesisStake,
     migration_sources: Vec<SnapshotSource>,
     migration_tx_data: Option<MigrationTxData>,
+    delegator: Option<String>,
```

Comment:
This can definitively be of type `IotaAddress`

Reply:
done